base/docs/helpdb/Base.jl

```diff
@@ -6416,7 +6416,7 @@ ctranspose!
 """
     object_id(x)
 
-Get a unique integer id for `x`. `object_id(x)==object_id(y)` if and only if `is(x,y)`.
+Get a hash value for `x` based on object identity. `object_id(x)==object_id(y)` if `x === y`.
```

Comment:
run `make docs` and commit the rst, or it'll confuse someone else as an unrelated change to whatever they're working on